lib/literature/helpers/meta_tag_helpers.ex

```diff
@@ -92,4 +92,45 @@ defmodule Literature.MetaTagHelpers do
       value -> value || tags[default_key] || @metatags[key]
     end
   end
+
+  @doc """
+  Render pagination link tags
+  """
+  def render_pagination_link_tags(
+        %{
+          live_action: :index,
+          page: %{page_number: page_number, total_pages: total_pages}
+        },
+        current_url
+      ) do
+    cond do
+      page_number == 1 and total_pages > 1 ->
```

Comment:
@rcmonsayac can we move these into smaller fucntions to make it more readable instead of cond

Reply:
@dgross881 updated